### PR TITLE
feat(footer)

### PR DESCRIPTION
**Pull request by MaxiV95**
image

### CODE REVIEW

**Comment by MaxiV95** [comment]
Ready for final review!

File: src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+					<LinkedinCircleIcon color="#999999" />
+					<WhatsappCircleIcon color="#999999" />
+					<FacebookCircleIcon color="#999999" />
+					<InstagramCircleIcon color="#999999" />
+				</span>
+			</div>
+			<div>
+				<span>
+					<MailIcon color="#999999" />
+					<p>+5119999999999</p>
```
Comment (manduinca) [inline src/components/footer/Footer.jsx]: +51 997 336 060

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+				</span>
+				<span>
+					<PhoneIcon color="#999999" />
+					<p>email@contact.com</p>
```
Comment (manduinca) [inline src/components/footer/Footer.jsx]: miguel.roncal@marc.com.pe

**Review by manduinca — CHANGES_REQUESTED** [review]
Update the data, please.

**Review by SebaSatelier — APPROVED** [review]

**Comment by MaxiV95** [comment]
Updated!

image

**Review by manduinca — APPROVED** [review]

**Pull request merged**